packages/f-registration/src/components/Registration.vue

```diff
@@ -130,6 +154,19 @@ import tenantConfigs from '../tenants';
 import RegistrationServiceApi from '../services/RegistrationServiceApi';
 import EventNames from '../event-names';
 
+/**
+ * Tests for existence of valid chars only in a string.
+ *
+ * @param {string} value The string to test.
+ * @return {boolean} True if there are no invalid chars in value, false otherwise.
+ */
+const validCharsInName = value => {
+    if (typeof value === 'undefined' || value === null || value === '') {
+        return true;
+    }
+    return /^[\u0060\u00C0-\u00F6\u00F8-\u017Fa-zA-Z-' ]*$/.test(value);
```

Comment:
Probably should exclude the multiplication sign ('×' = `\u00D7`) here if we're disallowing the divide sign at `\u00f7`.

`/^[\u0060\u00C0-\u00F6\u00F8-\u017Fa-zA-Z-' ]*$/.test('×')` => `true`

```suggestion
    return /^[\u0060\u00C0-\u00D6\u00D8-\u00F6\u00F8-\u017Fa-zA-Z-' ]*$/.test(value);
```

Reply:
I am using feature parity with ConsumerWeb production so we can do like for like testing.

Reply:
Cool, will resolve this for now and query in our team channel.